src/App.tsx

```diff
@@ -1,173 +1,70 @@
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import { People } from './components/People';
+
+export const App = () => {
+  return (
+    <HashRouter>
+      <div data-cy="app">
+        <nav
+          data-cy="nav"
+          className="navbar is-fixed-top has-shadow"
+          role="navigation"
+          aria-label="main navigation"
+        >
+          <div className="container">
+            <div className="navbar-brand">
+
+              <NavLink
+                exact
+                to="/"
+                className="navbar-item"
+                activeClassName="navbar-item has-background-grey-lighter"
+              >
+                Home
+              </NavLink>
+
+              <NavLink
+                to="/people"
+                className="navbar-item"
+                activeClassName="navbar-item has-background-grey-lighter"
+              >
+                People
+              </NavLink>
+
+            </div>
+          </div>
+        </nav>
+
+        <main className="section">
+          <div className="container">
+            <Switch>
+              <Route exact path="/">
```

Comment:
I would rather suggest that you to split this render on routes with separate /pages folder structure to be e.g. :

`/pages/HomePage.tsx, /pages/NotFoundPage.tsx, /pages/PeoplePage.tsx`

and pass them on to proper routes

you have described this in task requirements:

"Create the HomePage available at / with just a title Home Page
Create the PeoplePage available at /people with a title People Page
Implement NotFoundPage with a title Page not found that is shown for all the other URLs;"